profile/README.md

```diff
@@ -1,60 +1,89 @@
+# Resources
 
-It will be up to wallet software to display this information to the users, but by making the Flow transaction format easy to statically analyze, we create the possibility for a more transparent transaction approval process.
+**Developer Resources**
 
-### Smart User Accounts: no more seed words or lost keys  
+- [Developer Portal](https://developers.flow.com)
+- [Flow CLI](https://developers.flow.com/tools/flow-cli): The Flow CLI is Flow’s primary development and testing environment, similar to Hardhat and Foundry for developers familiar with writing in Solidity.
+- **Clients:**
+  - [Flow GO SDK](https://github.com/onflow/flow-go-sdk)
+  - [Flow Client Library](https://developers.flow.com/tools/clients/fcl-js) (JS, SWIFT, iOS, Python)
+  - [Flow SDK for Unity](https://developers.flow.com/tools/clients/unity-sdk)
 
-Flow is designed with flexibility in mind. Over the past year, Dapper Labs has pioneered a variety of usability enhancements to the Ethereum account model as part of the Dapper Smart Contract Wallet. Those enhancements are part of the native account model on Flow:  
 
-*   Optional, modular, smart contract functionality built into every Flow wallet
-*   This supports automated processes or more sophisticated authorization controls, in turn enabling good user experience. For example, dapps can easily make sure consumers never lose their assets – or access to their accounts – with secure account recovery flows
-*   Added security through optional multiple signature support, with the ability to cycle out old keys regularly to avoid security leaks
+- [**Wallets**](https://developers.flow.com/community-resources/wallets)**:** Flow has several wallets to choose from, including the Flow Reference Wallet that any developer can fork and extend to build their own.
+
+**Learn more about:**
+
+- [Roadmap](https://flow.com/flow-roadmap)
+- [Account Abstraction on Flow](https://flow.com/account-abstraction)
+- [Account linking](https://flow.com/account-linking)
+- [EVM on Flow](https://forum.flow.com/t/evm-on-flow-beyond-solidity/5260)
+- [Flow Reference Wallet](https://frw.gitbook.io/flow-reference-wallet/)
+- [Bridging](https://flow.com/use-flow/bridges)
+- [Comparison with other blockchains](https://developers.flow.com/build/building-vs-other-chain)
+
+**Community channels:**
+
+- [Discord](https://discord.gg/flow)
+- [Twitter](https://twitter.com/flow_blockchain)
+- [Telegram](https://t.me/flow_community)
+
+**Community:**
+
+- [Flowdiver](https://flowdiver.io): the Flow block explorer
+- [Flowverse](https://www.flowverse.co/): a community-run ecosystem portal
+- [Emerald City DAO](https://ecdao.org): a community of Flow builders
+- [Increment.fi](https://increment.fi): Swap, lend, stake and more 
+- [Flowty](https://flowty.io): NFT Lending and Renting
```

Comment:
We should add:
- [Evaluate Market] (evaluate.xyz): NFT Trading

Reply:
done!